common/build/build-common/api-extractor-lint.json

```diff
@@ -52,10 +52,10 @@
 	},
 	// Bundle local dependencies so we can validate cross-package relationships
 	"bundledPackages": [
-		"@fluidframework/.*",
-		"@fluid-internal/.*",
-		"@fluid-experimental/.*",
-		"@fluid-private/.*",
-		"@fluid-tools/.*"
+		"@fluidframework/*",
+		"@fluid-internal/*",
+		"@fluid-experimental/*",
+		"@fluid-private/*",
+		"@fluid-tools/*"
```

Comment:
Nope. This only impacts how api-extractor decides which packages it should include in its bundling, all of which is done locally.

Reply:
Oh, sorry, I misunderstood the question. The answer is still no, but that's because this file is consumed strictly by file-path reference everywhere - we don't use package deps in our api-extractor configs.